DESCRIPTION.md

```diff
@@ -9,15 +9,15 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 # Release Notes
 
 - v3.0.5(TBD)
-  - Added a feature that lets you add connection definitions to the `config.toml` configuration file. A connection definition refers to a collection of connection parameters. The connection configuration name must begin with **connections**, similar to the following that defines the parameters for the `prod` connection:
+  - Added a feature that lets you add connection definitions to the `connections.toml` configuration file. A connection definition refers to a collection of connection parameters, similar to the following that defines the parameters for the `prod` connection:
```

Comment:
The wording might leave the impression that `prod` is special.  What if it said something like:

"...a collection of connection parameters, for example, if you wanted to define a connection named `prod` :"